tests/acceptance/acceptance_test.go

```diff
@@ -80,6 +80,16 @@ var _ = Describe("Acceptance", func() {
 			// not register as running, even though all child resources are fine
 			// :shrug_emoji:
 			Expect(podList.Items[0].Spec.Containers[0].Image).To(Equal(nginxImage))
+
+			Expect(kClient.Delete(context.Background(), &pSub)).To(Succeed())
+
+			Eventually(func() int {
+				podList = &v1.PodList{}
+				err := kClient.List(context.Background(), podList, opts...)
+				Expect(err).NotTo(HaveOccurred())
+				return len(podList.Items)
+			}, 5*time.Minute, 10*time.Second).Should(Equal(0))
```

Comment:
could we either have this:
a) in an after each
b) in a separate test (this one talks about deploying a thing)
c) in this test which is renamed with some `By`s to make steps clear

Reply:
I prefer c)